--- origsrc/libqtxdg-1.2.0/qiconfix/qiconloader_p.h	2015-03-29 04:30:27.000000000 -0500
+++ src/libqtxdg-1.2.0/qiconfix/qiconloader_p.h	2015-04-28 18:29:03.483118800 -0500
@@ -57,6 +57,12 @@
 #include <QtCore/QVector>
 #include <QtCore/QTypeInfo>
 
+#ifdef QTXDG_COMPILATION
+    #define QTXDG_PRIVATE_API    Q_DECL_EXPORT
+#else
+    #define QTXDG_PRIVATE_API    Q_DECL_IMPORT
+#endif
+
 //QT_BEGIN_NAMESPACE
 
 namespace QtXdg {
@@ -154,7 +160,7 @@ private:
     bool m_valid;
 };
 
-class Q_GUI_EXPORT QIconLoader
+class QTXDG_PRIVATE_API QIconLoader
 {
 public:
     QIconLoader();
